packages/block-editor/src/components/gradients/index.js

```diff
@@ -45,7 +45,7 @@ export function __experimentalUseGradient( {
 			customGradient: attributes[ customGradientAttribute ],
 			gradients: getSettings().gradients,
 		};
-	}, [ clientId ] );
+	}, [ clientId, gradientAttribute, customGradientAttribute ] );
```

Comment:
Were this missed before @jorgefilipecosta

Reply:
Yes I missed them, thank you for the fix 👍